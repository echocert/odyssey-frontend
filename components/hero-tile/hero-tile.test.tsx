import React from "react";
import { render, screen } from '@testing-library/react';
import HeroTile from "./hero-tile";
import { create } from "react-test-renderer";

describe("render Hero tile inside main page", () => {
  it("Verify the snapshot of Hero tile is matched", () => {
    const wrap = create(<HeroTile />).toJSON();
    expect(wrap).toMatchSnapshot();
  });
});