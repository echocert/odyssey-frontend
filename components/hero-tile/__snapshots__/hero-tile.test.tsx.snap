// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Hero tile inside main page Verify the snapshot of Hero tile is matched 1`] = `
<div
  className="hero-tile"
>
  <div
    className="hero-tile__item hero-tile__item-one"
  >
    <label
      className="hero-tile__header"
    >
      We Make Process Simple
    </label>
    <p
      className="hero-tile__info"
    >
      Try the Free Odyssey Scan Solution Today
    </p>
  </div>
  <div
    className="hero-tile__item hero-tile__item-two"
  >
    <div
      className="hero-tile__item-list"
    >
      <img
        alt=""
        className="hero-tile__item-image"
        src="../assets/images/scan.png"
      />
      <label
        className="hero-tile__item-header"
      >
        Scan
      </label>
      <p
        className="hero-tile__item-list-item-one hero-tile__item-list-item-one-first-paragraph"
      >
        Easily generate an audit link for relevant files
      </p>
      <p
        className="hero-tile__item-list-item-one hero-tile__item-list-item-one-second-paragraph"
      >
        Accessible 24/7
      </p>
    </div>
  </div>
  <div
    className="hero-tile__item hero-tile__item-three"
  >
    <div
      className="hero-tile__item-list"
    >
      <img
        alt=""
        className="hero-tile__item-image"
        src="../assets/images/organize.png"
      />
      <label
        className="hero-tile__item-header"
      >
        Organize
      </label>
      <p
        className="hero-tile__item__list-item-two"
      >
        Secure & Searchable
      </p>
      <p
        className="hero-tile__item__list-item-two"
      >
        AUDIT Ready
      </p>
    </div>
  </div>
  <div
    className="hero-tile__item hero-tile__item-four"
  >
    <div
      className="hero-tile__item-list"
    >
      <img
        alt=""
        className="hero-tile__item-image"
        src="../assets/images/store.png"
      />
      <label
        className="hero-tile__item-header"
      >
        Store
      </label>
      <p
        className="hero-tile__item__list-item-three"
      >
        Free and unlimited file storage
      </p>
      <p
        className="hero-tile__item__list-item-three"
      >
        Securely stored on Azure by Microsoft
      </p>
    </div>
  </div>
</div>
`;
